js/interface_common.js

```diff
@@ -819,36 +832,109 @@ function replyTextKeypress(e) {
                     }
                     $tas = tweetForm.find("textarea");
                     splitedPostsCount = $tas.length;
+                    caretPos = -1;
+                    if (icurrentta >= i && icurrentta > 0) {
+                        icurrentta -= 1;
+                        cci = getPostSpittingCI(icurrentta);
+                    }
+                    var $targetta = $($tas[icurrentta]);
                 }
             }
-            if (typeof($newta) !== 'undefined' && $newta[0] !== document.activeElement)
-                $newta.focus();
 
-            c = 140 - $tas[$tas.length - 1].value.length - (2 * splitedPostsCount.toString().length) - 4;
-            if (typeof(reply_to) !== 'undefined' &&
-                !checkPostForMentions($tas[$tas.length - 1].value, reply_to, 140 - c - reply_to.length))
-                c -=  reply_to.length;
+            if (typeof($targetta) !== 'undefined' && $targetta[0] !== document.activeElement) {
+                $this = $targetta;
+                $this.focus();
+                $this.caret(caretPos);
+            }
         }
-        var remainingCount = tweetForm.find(".post-area-remaining");
+    }
 
-        if( c < 0 )
-            remainingCount.addClass("warn");
-        else
-            remainingCount.removeClass("warn");
+    function getPostSplitingPML() {
+        var pml = 140 -(i+1).toString().length -splitedPostsCount.toString().length -4;
 
-        if (usePostSpliting && !$this.parents('.directMessages').length)
-            remainingCount.text(splitedPostsCount.toString() + ". post: " + c.toString());
-        else
-            remainingCount.text(c.toString());
+        // if mention exists, we shouldn't add it while posting.
```

Comment:
it is not my comment. I suppose there is should be option for this. we can remove attribute `data-reply-to` from `$newta` optionally and so mentions will appear only in first post of chain.